File: javascript/fundamentals/polymorphism.js
class calc{
    add(){
        console.log("onside nomarg method");
    }
    add(num){
        console.log("inside one arg method");
    }
    add(num1,num2){
        console.log("inside two arg method");
    }
}


var cal=new calc()
cal.add(10,20);
cal.add()

File: objectorientedprogramming/oops/polymorphism.py


#method overloading

class Maths:
    def add(self):
        print("inside no arg")

    def add(self,num1):
        print("onside one arg")

    def add(self,num1,num2):
        print("inside two arg")

m=Maths()
m.add(10,20)

#method overriding